tests/fixture-monorepo/packages/one/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"compilerOptions": {
+		"jsx": "react",
+		"moduleResolution": "node",
+		"composite": true,
+		"outDir": "dist"
+	},
+	"include": ["src/index.ts"]
```

Comment:
One issue I fixed here is that you didn't include `name.ts` in the `include. According to the `composite` docs, all files must be matched: https://www.typescriptlang.org/tsconfig#composite

However, the bug still persists:
`packages/one/src/name.ts' is not listed within the file list of project ''. Projects must list all files or use an 'include' pattern.`

I'm thinking this has to do with rollupd-plugin-dts not passing in the tsconfig's `includes` which is necessary for `composite`.

Reply:
Ooops. I thought I had cleaned everything up. Yes, that was supposed to be `src/**/*` after the addition of the second file, and I would not be surprised if you are correct.